Mathlib/Tactic/Relation/Main.lean

```diff
@@ -0,0 +1,29 @@
+/-
+Copyright (c) 2022 Siddhartha Gadgil. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Siddhartha Gadgil
+-/
+
+import Lean
+
+open Lean Meta
+
+/--
+matches to expressions of the form `r x y` with `r` a relation and returns the triple `(r, x, y)` if there is a match. Note that `r` may be obtained applying a function to arguments.
```

Comment:
Line is too long. (Unfortunately at the moment the linter only runs if you push from a branch of the main repository.)